x/ibc/09-localhost/types/codec.go

```diff
@@ -27,17 +28,10 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 }
 
 var (
-	amino = codec.New()
```

Comment:
we don't need this because 09-localhost does not internal amino encoding, just registration is needed to decode/encode client states at genesis and exporting genesis